Should be able to specify start-time and end-time

The cli `--end-time` argument doesn't seem to work and returns the following error:

```
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/pandas/core/arrays/datetimes.py", line 2646, in _infer_tz_from_endpoints
    inferred_tz = timezones.infer_tzinfo(start, end)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "timezones.pyx", line 369, in pandas._libs.tslibs.timezones.infer_tzinfo
AssertionError: Inputs must both have the same timezone, None != tzoffset(None, -61200)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/<redacted>/Transit-Simulator/main.py", line 284, in <module>
    rng = pd.date_range(start=start_datetime, end=end_datetime, freq='30s')
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/pandas/core/indexes/datetimes.py", line 1008, in date_range
    dtarr = DatetimeArray._generate_range(
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/pandas/core/arrays/datetimes.py", line 445, in _generate_range
    tz = _infer_tz_from_endpoints(start, end, tz)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/pandas/core/arrays/datetimes.py", line 2649, in _infer_tz_from_endpoints
    raise TypeError(
TypeError: Start and end cannot both be tz-aware with different timezones
```

--- a/main.py
+++ b/main.py
@@ -81,7 +81,7 @@ else:
     simulation_start_time = current_time.strftime("%H:%M:%S")
 
 if args.end_time:
-    simulation_end_time = args.date
+    simulation_end_time = args.end_time
 else:
     start_time_datetime = datetime.strptime(simulation_start_time, "%H:%M:%S")
     new_time_datetime = start_time_datetime + timedelta(hours=1)

